src/nl/surf/eduhub_rio_mapper/rio/loader.clj

```diff
@@ -31,10 +31,16 @@
   {:pre [element]}
   (= "true" (single-xml-unwrapper element "ns2:requestGoedgekeurd")))
 
+(defn log-rio-call [msg element]
+  (log/infof (format "RIO %s; SUCCESS: %s" msg (goedgekeurd? element))))
```

Comment:
Logging a rio action response, right?  Then rename it to `log-rio-action-response`.

This is for debugging, right?  Then use level "debug".

Those responses a pretty small, right?  The just log the entire element, that way you don't need all the ugly if/let stuff here below in `rio-resolve-response`.

Reply:
The element can be bigger than what I want to log.

But I've cleaned up `rio-resolver-response`.